domains.txt

```diff
@@ -746,6 +833,8 @@ webrecruitglobal.com
 welovesalt.com
 wenhamcarter.com
 westbourne-partners.com
+whitehallresources.co.uk
```

Comment:
Whitehall Resources was founded with one goal – to be the biggest and best recruiter globally, in the markets we deliver to.